src/main/java/Gutti/FindCommand.java

```diff
@@ -0,0 +1,74 @@
+package Gutti;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a command that searches for tasks with descriptions containing a specified word.
+ * <p>
+ * This class extends {@code Command} and provides functionality to search through a list of tasks
+ * and output tasks that match the search criteria.
+ * </p>
+ */
+public class FindCommand extends Command{
```

Comment:
Good naming conventions used  and JavaDoc used where appropriate. Code is concise and to the point.